packages/telegraf/extra/dcos-telegraf.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Telegraf: collects and reports metrics
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+
+User=dcos_telegraf
+PermissionsStartOnly=True
+
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/telegraf.env
+
+LimitNOFILE=16384
+
+ExecStartPre=/opt/mesosphere/bin/bootstrap ${SERVICE_ACCOUNT}
```

Comment:
I would change variable name for `${SERVICE}`. Service account creation is only of many possible things that `bootstrap` procedure does.